_source/no-search/p8s-remote-write.md

```diff
@@ -0,0 +1,93 @@
+
+Integrate your Prometheus toolkit with the Logz.io Metrics platform to continuously monitor your system’s health and help you maintain system performance, speed, and resilience, and address problematic trends when they first appear - and before they become critical and threaten crashes or system outages.
+
+![Showcased Metrics dashboard](https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana/metrics-intro.png)
+
+To send your Prometheus application metrics to a Logz.io Infrastructure Monitoring account, use remote write to connect to Logz.io as the endpoint. Your data is formatted as JSON documents by the Logz.io listener. 
+
+If you have multiple Prometheus server instances, you'll have to add Logz.io as an endpoint for each instance. 
+
+By default, all metrics in your Prometheus server(s) will be sent to Logz.io. To drop or send specific metrics, add Prometheus labeling before enabling the remote write, or as part of the remote write configuration. You can read more about Prometheus relabeling tricks <a href ="https://medium.com/quiq-blog/prometheus-relabeling-tricks-6ae62c56cbda" target="_blank">here. <i class="fas fa-external-link-alt"></i> </a> 
+
+* TBD: Set parallelism level for sending data - how many connections to open to the remote write listener. While the default is 1000, but we recommend configuring much fewer connections. 
+
+* TBD: This is set in the configuration file. Parameter for # of connections (more data --o--> more channels)
+  Sending data - best practice in configuring connection channels: 
+
+* TBD: If you have both Prometheus & Grafana, you can activate a dashboard as part of the remote write configuration that will show you the queue size and how many metrics you're sending. If your queue size increases, it might be necessary to open an additional channel. 
+
+You can read more about Prometheus Remote Write Tuning <a href ="https://prometheus.io/docs/practices/remote_write/" target="_blank">here. <i class="fas fa-external-link-alt"></i> </a> 
+
+Once your metrics are flowing, export your existing Prometheus and Grafana dashboards to Logz.io Infrastructure Monitoring as JSON files.  
+
+
+### Configuring Remote Write to Logz.io
+
+
+Within Prometheus:
+
+1. Define Logz.io as a new endpoint, using remote write.
+2. Provide the Logz.io metrics account token and listener address (URL).
+
+  - To find the default token in the [General Settings](https://app.logz.io/#/dashboard/settings/general) page , click **<i class="li li-gear"></i> > Settings > General** in the top menu.
+
+<video autoplay loop>
+    <source src="https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana-videos/p8sgo-to-acct_settings2.mp4" type="video/mp4"/>
+</video>
+
+  - To find the correct listener URL for your region, look [here](/user-guide/log-shipping/listener-ip-addresses.html). 
```

Comment:
can we make this link and the one above to be opened on a separate page? so the user will not leave the current instructions

Reply:
Absolutely. Done.